manuscript/teaching-reproducibility.Rmd

```diff
@@ -290,9 +290,30 @@ This example highlights the enormous cost
 associated with not adhering to reproducible practices
 and having workflows that are opaque and hard to review.
 
+As illustrated in the case we have highlighted above,
+reproducibility errors are not isolated to novices performing irrelevant data analyses.
+They are occurring at the highest level in both industry and academia
+and can have substantial real-world consequences.
+When selecting failures to share with students,
+it can be beneficial to focus on
+those that students can easily identify with,
+those that have had a big impact,
+and those that come from prolific researchers/companies.
+This is to illustrate that these errors can happen to anyone
+and emphasize that we are not sharing them cast blame,
+but to highlight the importance of reproducible practices on all levels.
+Another example that illustrates the previous points
```

Comment:
I am a bit lost as to what previous points this example is referring to... Not sure if this is because of the new addition, or if we need some more context or reorganization here.

Reply:
```suggestion
Another example of an impactful reproducibility mistake in from a well-known research team 
```